ext/hal/inventek/ism43362-m3g-l44/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Kconfig - Inventek wifi hal config
+
+# Copyright (c) 2016 Linaro Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_ISM43362_M3G_L44
+	bool
+	prompt "Use Inventek ism43362_m3G_l44 wifi component"
+	default n
+	select HAS_INVENTEK_WIFI
+	help
+	  To be used if Inventek ism43362_m3G_l44 wifi component is on the board
+
+config HAS_INVENTEK_WIFI
+    bool
+
+config WIFI_SPI_DEV_NAME
```

Comment:
Any Zephyr specific part for adapting the HAL to work, as noticed in other commit, should go - if possible - in drivers/wifi etc... It will then centralize all the configuration required to get the driver working.